### PR TITLE
skip rename assert on bz

### PR DESCRIPTION
**Pull request by pondrejk**

### CODE REVIEW

**Review by mirekdlugosz — APPROVED** [review]
LGTM

File: tests/foreman/sys/test_rename.py [hunk 1]
```diff
@@ -43,6 +44,8 @@ def test_positive_rename_satellite(self, module_org, module_product):
         :setup: Satellite server with synchronized rh and custom
             repos and with a registered host
 
+        :bz: 1901983
+
```
Comment (jameerpathan111) [inline tests/foreman/sys/test_rename.py:48]: @pondrejk the issue seems similar to https://bugzilla.redhat.com/show_bug.cgi?id=1897360

File: tests/foreman/sys/test_rename.py [hunk 2]
```diff
@@ -57,7 +60,7 @@ def test_positive_rename_satellite(self, module_org, module_product):
                 resync repos, republish CVs and re-register hosts
             7. Create new entities (run end-to-end test from robottelo)
 
-        :BZ: 1469466
+        :BZ: 1469466, 1897360
```
Comment (jameerpathan111) [inline tests/foreman/sys/test_rename.py:63]: there are two `:BZ:` docstring token for same test, on line no. 47 and 63.

Comment (pondrejk) [reply tests/foreman/sys/test_rename.py]: my apologies, good catch! Strange that pre-commit checks didn't catch this

**Comment by pondrejk** [comment]
filed https://github.com/SatelliteQE/robottelo/issues/8213

**Review by mshriver — APPROVED** [review]

**Pull request merged**